### PR TITLE
EVEREST-182 Labels for DatabaseCluster and DatabaseClusterBackup reso…

### PR DESCRIPTION
**Pull request by gen1us2k**
…urces

### CODE REVIEW

File: controllers/databasecluster_controller.go [hunk 1]
```diff
@@ -341,6 +341,15 @@ func (r *DatabaseClusterReconciler) Reconcile(ctx context.Context, req ctrl.Requ
...
 	_, ok := database.ObjectMeta.Annotations[restartAnnotationKey]
+	database.ObjectMeta.Labels = map[string]string{
+		databaseClusterNameLabel: database.Name,
+	}
+	if database.Spec.DataSource != nil {
+		database.ObjectMeta.Labels[fmt.Sprintf(backupStorageNameLabelTmpl, database.Spec.DataSource.ObjectStorageName)] = "used"
+	}
+	for _, schedule := range database.Spec.Backup.Schedules {
+		database.ObjectMeta.Labels[fmt.Sprintf(backupStorageNameLabelTmpl, schedule.ObjectStorageName)] = "used"
+	}
```
Comment (oksana-grishchenko) [inline controllers/databasecluster_controller.go:352]: Should we clean up labels when the ObjectStorageName is not used anymore?

File: controllers/databaseclusterbackup_controller.go [hunk 2]
```diff
@@ -88,7 +88,7 @@ func (r *DatabaseClusterBackupReconciler) Reconcile(ctx context.Context, req ctr
 	}
 	backup.ObjectMeta.Labels = map[string]string{
 		databaseClusterNameLabel: backup.Spec.DBClusterName,
-		backupStorageNameLabel:   backup.Spec.ObjectStorageName,
+		fmt.Sprintf(backupStorageNameLabelTmpl, backup.Spec.ObjectStorageName): "used",
```
Comment (oksana-grishchenko) [inline controllers/databaseclusterbackup_controller.go:91]: the constant is repeated 3 times, I think it's a sign to move it to a constant

**Review by oksana-grishchenko — APPROVED** [review]
Looks like a great idea, thanks for implementing this! 
Approving but please consider the comments:

File: controllers/databasecluster_controller.go (hunk 1 above)
Comment (gen1us2k) [reply controllers/databasecluster_controller.go:352]: We should. Great catch. Thanks

**Pull request merged**